src/frontend/searchAdvanced.ml

```diff
@@ -785,12 +785,10 @@ let search_page () =
                     )
                     ()
             )
-          ()
-      )
```

Comment:
?

Reply:
Was added by mistake in commit fb2537613eb96685a6f4061e09662b17dd8d7164

I wont compile because of it